### DISCUSSION TITLE
Newly created larger runner still uses old image version

### DISCUSSION THREAD

**Discussion by christang0807** [actions/virtual-environments#8617] [General questions] [answered]
### Description

We have a larger runner using ubuntu-22.04 latest but while running it in our workflows it is actually still running with the following version:
  `Version: 20230924.1.0`
this produces an issue with the specific version of Chrome and Cypress so we cannot run our E2E tests unless we use workarounds like installing the latest version of Chrome on the machine or not using a larger runner and just using the default `ubuntu-22.04-latest`.

### Platforms affected

- [ ] Azure DevOps
- [ ] GitHub Actions - Standard Runners
- [X] GitHub Actions - Larger Runners

### Runner images affected

- [ ] Ubuntu 20.04
- [X] Ubuntu 22.04
- [ ] macOS 11
- [ ] macOS 12
- [ ] macOS 13
- [ ] Windows Server 2019
- [ ] Windows Server 2022

### Image version and build link

Runner Image
  Image: ubuntu-22.04
  Version: 20230924.1.0

### Is it regression?

No

### Expected behavior

I expected our larger runner to be using `Image Version: 20231016.1.0` (current version) where Chrome has been updated and no longer experiences issues with Cypress.

### Actual behavior

Our larger runner is still using this:
```Runner Image
  Image: ubuntu-22.04
  Version: 20230924.1.0
```
### Repro steps

1. Create/provision a large runner with ubuntu-22.04 latest
2. Run a workflow using the new larger runner
3. In the Set up job, an older version of the image is actually run instead.

**Comment by mikhailkoliada** [comment, marked as answer]
Hello, it is expected as we were not updating image version on larger runners to solve some software issues, new version should be available today or on Monday I believe

**Reply by christang0807** [reply]
Is this a known disconnect between the standard runners and large runners where larger runners just lag behind in versions? And how would we know when a larger runner will get the updated version?
